src/kafka/protocol/kafka_primitives.hh

```diff
@@ -94,6 +94,10 @@ public:
         return error::kafka_error_code::get_error(_value);
     }
 
+    [[nodiscard]] const error::kafka_error_code* operator->() const noexcept {
+        return &error::kafka_error_code::get_error(_value);
```

Comment:
Safe to get reference to the return value of `get_error` as `get_error` returns a reference to static const variable!

Reply:
it is a pointer nit a reference tho